shared/util/set-notifications.js

```diff
@@ -1,55 +0,0 @@
-/* @flow */
-
-import engine from '../engine'
-import {notifyCtlSetNotificationsRpc} from '../constants/types/flow-types'
-
-type NotificationChannels = {
```

Comment:
this pattern was very problematic. the original idea (mine i think) was we could call this function and it'd have its own local copy and it'd flip the bits to true and call the setNotifications. This runs into timing issues if we've already connected and sent this, then change it later. it's just mentally simpler to set this once and do it when we connect once (its not like this is dynamic in any way ultimately)